### DISCUSSION TITLE
How to update upgrade find packages and install command

### DISCUSSION THREAD

**Discussion by LukaszRobertKlimkiewicz** [nix-community/nix-on-droid#310] [Q&A]
Can you please tell me how to update upgrade find packages and install using a command.

**Comment by t184256** [comment]
Installing packages with some `install` commands is a last-century concept alien to Nix. Instead [edit a configuration file](https://nix-community.github.io/nix-on-droid/nix-on-droid-options.html#sec-usage) and execute `nix-on-droid switch`. The option you're interested in is, probably, [environment.packages](https://nix-community.github.io/nix-on-droid/nix-on-droid-options.html#opt-environment.packages). Upgrading is, similarly, a non-concept for Nix.

Finding packages is done with `nix search nixpkgs SOMETHING`. That'd be slow.

Please familiarize yourself with Nix documentation for further details.
